src/actions/index.js

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+import _ from 'lodash';
+
+export const GET_PRODUCTS = "GET_PRODUCTS";
+
+let selectedOptions = {};
```

Comment:
Is there a way to do this without side-effects (updating variable outside of function)?